types/parse/index.d.ts

```diff
@@ -1016,6 +1016,7 @@ declare global {
             logIn<T extends User>(username: string, password: string, options?: FullOptions): Promise<T>;
             logOut<T extends User>(): Promise<T>;
             requestPasswordReset<T extends User>(email: string, options?: SuccessFailureOptions): Promise<T>;
+            requestEmailVerification<T extends User>(email: string, options?: SuccessFailureOptions): Promise<T>;
```

Comment:
```suggestion
            requestEmailVerification<T extends User>(email: string, options?: RequestOptions): Promise<T>;
```

Reply:
Thank you @RaschidJFR. Suggestion committed.